tap_platformpurple/streams/base.py

```diff
@@ -54,21 +52,17 @@ def sync_data(self):
         while not done:
             max_date = start_date
 
-            LOGGER.info(
-                "Querying {} starting at {}".format(table, start_date))
+            LOGGER.info("Querying {} starting at {}".format(table, start_date))
 
             body = {
-                'startMSeconds': int(start_date.timestamp() * 1000),
-                'endMSeconds': int(end_date.timestamp() * 1000)
+                "startMSeconds": int(start_date.timestamp() * 1000),
+                "endMSeconds": int(end_date.timestamp() * 1000),
             }
 
             if filters is not None:
-                body['filters'] = filters
+                body["filters"] = filters
```

Comment:
I'm assuming this is a change `black` made. I'd be interested to know why it does this